src/i_oalsound.c

```diff
@@ -378,12 +376,12 @@ static void ResetParams(void)
     alListeneriv(AL_ORIENTATION, default_orientation);
     if (oal->EXT_EFX)
     {
-        alListenerf(AL_METERS_PER_UNIT, 1.0f / OAL_MAP_UNITS_PER_METER);
+        alListenerf(AL_METERS_PER_UNIT, OAL_METERS_PER_MAP_UNIT);
     }
 
     // Context state parameters.
-    alDistanceModel(AL_INVERSE_DISTANCE_CLAMPED); // OpenAL 1.1 Specs, 3.4.2.
-    alSpeedOfSound(OAL_SPEED_OF_SOUND * OAL_MAP_UNITS_PER_METER);
+    alDistanceModel(AL_EXPONENT_DISTANCE_CLAMPED); // OpenAL 1.1 Spec, 3.4.6.
```

Comment:
Why this change (inverse distance to exponent distance)?

Reply:
I liked the curve response of the exponential model but the inverse model is fine and probably more performant ([lerp](https://github.com/kcat/openal-soft/blob/8f2fe93faf2752e99c49565b284437c527749f88/alc/alu.cpp#L1578) vs [pow](https://github.com/kcat/openal-soft/blob/8f2fe93faf2752e99c49565b284437c527749f88/alc/alu.cpp#L1622)). I changed it back to inverse here: https://github.com/fabiangreffrath/woof/pull/2215/commits/c06eb59a4beb79503737f9a333bee8c478db02af

They're pretty close:

![Sound Curves(1)](https://github.com/user-attachments/assets/5671a03b-7347-4a0e-8027-cfe90c4d9e17)